doc/intro.md

```diff
@@ -1,3 +1,46 @@
 # Introduction to witan.gwyn
 
-...coming soon
+## Model outline
+
+
+`witan.gwyn` aims to produce fire risk assessment scores as laid out on this flowchart:
+
+![lfb-model-flowchart](images/LFB_Model_Flowchart.png)
+
+
```

Comment:
Should we include the image which omits the step we haven't included or do we need a note saying what is currently complete?